x/mongo/driver/topology/server.go

```diff
@@ -82,9 +82,9 @@ func connectionStateString(state int64) string {
 
 // Server is a single server within a topology.
 type Server struct {
+	connectionstate int64
```

Comment:
Can we leave a comment here like [the existing ones](https://github.com/mongodb/mongo-go-driver/blob/master/x/mongo/driver/topology/connection.go#L41-L43) for atomically-accessed integers? That way, we'll be less likely to place another field on top in the future.

Reply:
Will add.